tests/js/server/aql/aql-index-hints.js

```diff
@@ -69,10 +77,45 @@ function indexHintSuite() {
       alternateEqualityIndex = 'skip_a';
       defaultSortingIndex = 'hash_a';
       alternateSortingIndex = 'skip_a_b';
+
+      internal.db._drop(cn2);
+      collection = internal.db._create(cn2);
+      collection.ensureIndex({type: "persistent", fields: ["value1"], name: persistentIdxName});
+      collection.ensureIndex({type: "persistent", fields: ["value2"], name: persistentIdxName2});
+      collection.ensureIndex({type: "geo", geoJson: true, fields: ["geo"], name: geoIdxName});
+      collection.ensureIndex({type: "geo", geoJson: true, fields: ["otherGeo"], name: geoIdxName2});
+      analyzers.save("geo_json", "geojson", {type: "point"}, ["frequency", "norm", "position"]);
+      let commonInvertedIndexMeta = {
+        type: "inverted", name: invertedIdxName, includeAllFields: true, fields: [
+          {"name": "geo", "analyzer": "geo_json"}
+        ]
+      };
+      collection.ensureIndex(commonInvertedIndexMeta);
+      commonInvertedIndexMeta = {
+        type: "inverted", name: invertedIdxName2, includeAllFields: true, fields: [
+          {"name": "otherGeo", "analyzer": "geo_json"}
+        ]
+      };
+      collection.ensureIndex(commonInvertedIndexMeta);
```

Comment:
arangosearch geo in 3.10 contains distance bug.
It fixed in devel but not in 3.10, I will backport it with new analyzer.
Please comment out it, and todo for me

Reply:
Hmm but this bug probably not affect your PR